src/utils/command.rs

```diff
@@ -1,43 +1,44 @@
 use std::{fmt::Display, io::Write};
 
-use crate::{execute, queue, write_cout};
+use crate::{execute, queue};
 
 use super::error::Result;
 
-/// A command is an action that can be performed on the terminal.
+/// An interface for a command that can be executed on the terminal.
 ///
-/// crossterm already delivers a number of commands.
-/// There is no need to implement them yourself.
-/// Also, you don't have to execute the commands yourself by calling a function.
+/// Crossterm provides a set of commands,
+/// and there is no immediate reason to implement a command yourself.
+/// In order to understand how to use and execute commands,
+/// it is recommended that you take a look at [Command Api](../#command-api) chapter.
 pub trait Command {
     type AnsiType: Display;
 
-    /// Returns the ANSI code representation of this command.
-    /// You can manipulate the terminal behaviour by writing an ANSI escape code to the terminal.
-    /// You are able to use ANSI escape codes only for windows 10 and UNIX systems.
+    /// Returns an ANSI code representation of this command.
+    /// An ANSI code can manipulate the terminal by writing it to the terminal buffer.
+    /// However, only Windows 10 and UNIX systems support this.
     ///
-    /// **This method is mainly used internally by crossterm!**
+    /// **This method is used internally by crossterm, and should not be called manually!**
     fn ansi_code(&self) -> Self::AnsiType;
 
     /// Execute this command.
     ///
-    /// On operating systems that do not support ANSI escape codes ( < Windows 10) we need to call WinApi to execute this command.
+    /// Windows versions lower than windows 10 do not support ANSI escape codes,
+    /// therefore a direct WinAPI call is made.
     ///
-    /// **This method is mainly used internally by crossterm!**
+    /// **This method is used internally by crossterm, and should not be called manually!**
```

Comment:
Okay, I will change it to "There is no need to call this method manually", I just want to prevent users to get the idea that they have to execute commands them selfs. And I want to point them in the direction of the execute/queue methods.

Reply:
This is a public API and has to be like this because macros are public and call those methods from outside the crate.